jerry-port/default/default-module.c

```diff
@@ -20,10 +20,15 @@
 #include <stdarg.h>
 #include <stdlib.h>
 #include <string.h>
+#include <sys/stat.h>
```

Comment:
Is this win32 compatible?

Reply:
Yes, it is. Windows has sys/stat.h, but no S_ISDIR macro, that's why we should define it.